src/lecturer/components/createQuestionView/CreateQuestionView.tsx

```diff
@@ -16,7 +16,7 @@ import AddIcon from '@material-ui/icons/Add';
 import DeleteIcon from '@material-ui/icons/Delete';
 import clsx from 'clsx';
 import { Location } from 'history';
-import { ChangeEvent, FormEvent, useContext, useRef, useState } from 'react';
+import { ChangeEvent, FormEvent, useCallback, useContext, useEffect, useRef, useState } from 'react';
```

Comment:
```suggestion
import { ChangeEvent, useCallback, useContext, useEffect, useRef, useState } from 'react';
```
You've removed usage of `FormEvent` from line 276 so you should delete import as well